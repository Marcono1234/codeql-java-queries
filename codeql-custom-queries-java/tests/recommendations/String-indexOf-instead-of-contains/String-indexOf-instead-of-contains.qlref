./recommendations/String-indexOf-instead-of-contains.ql
