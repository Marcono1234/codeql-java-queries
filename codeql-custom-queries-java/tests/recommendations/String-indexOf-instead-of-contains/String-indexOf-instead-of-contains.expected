| Test.java:4:13:4:40 | ... == ... | Should use !String.contains(...) instead |
| Test.java:5:13:5:40 | ... != ... | Should use String.contains(...) instead |
| Test.java:6:13:6:39 | ... > ... | Should use String.contains(...) instead |
| Test.java:7:13:7:39 | ... < ... | Should use String.contains(...) instead |
| Test.java:8:13:8:40 | ... <= ... | Should use !String.contains(...) instead |
| Test.java:9:13:9:40 | ... >= ... | Should use !String.contains(...) instead |
| Test.java:10:13:10:39 | ... >= ... | Should use String.contains(...) instead |
| Test.java:11:13:11:39 | ... <= ... | Should use String.contains(...) instead |
| Test.java:12:13:12:38 | ... < ... | Should use !String.contains(...) instead |
| Test.java:13:13:13:38 | ... > ... | Should use !String.contains(...) instead |
| Test.java:14:13:14:44 | ... == ... | Should use !String.contains(...) instead |
| Test.java:15:13:15:44 | ... != ... | Should use String.contains(...) instead |
| Test.java:16:13:16:43 | ... > ... | Should use String.contains(...) instead |
| Test.java:17:13:17:44 | ... <= ... | Should use !String.contains(...) instead |
| Test.java:18:13:18:43 | ... >= ... | Should use String.contains(...) instead |
| Test.java:19:13:19:42 | ... < ... | Should use !String.contains(...) instead |
